### PR TITLE
[#155322114] Add Asset Category Model 

### PR DESCRIPTION
**Pull request by HawiCaesar**
### What does this PR do ?
-  Add the asset category model

### How can this be manually tested ?
`python manage.py test`

### Description of work done
- Create a asset category model
- Create tests for the asset category model

### Relelvant PT story
[#155322114](https://www.pivotaltracker.com/story/show/155322114)

### CODE REVIEW

File: art/api/models.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
-# Create your models here.
+
+class AssetCategory(models.Model):
+    """ Stores all asset categories """
+    name = models.CharField(max_length=40, null=False)
```
Comment (egichuri) [inline art/api/models.py]: Can we use something like `category_name`? `name` doesn't sound descriptive enough :)

File: art/api/models.py [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+    created_at = models.DateTimeField(auto_now_add=True, editable=False)
+    modified_at = models.DateTimeField(auto_now=True, editable=False)
```
Comment (egichuri) [inline art/api/models.py]: `last_modified`

File: art/api/tests/test_asset_category_model.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class AssetCategoryModelTest(TestCase):
+    """ Tests for the Asset Category Model """
+    def setUp(self):
+        AssetCategory.objects.create(name="Accessories")
+        self.category = AssetCategory.objects.get(name="Accessories")
+
+    def test_can_save_a_category(self):
+        self.assertIn("Accessories", self.category.name)
```
Comment (egichuri) [inline art/api/tests/test_asset_category_model.py]: we might need to create a new category here and check for an increase in the category count.

File: art/api/models.py (hunk 1 above)
Comment (HawiCaesar) [reply art/api/models.py]: I see your point. I will change this.

File: art/api/tests/test_asset_category_model.py (hunk 3 above)
Comment (HawiCaesar) [reply art/api/tests/test_asset_category_model.py]: Alright makes sense! I have changed this.

File: art/api/models.py (hunk 2 above)
Comment (HawiCaesar) [reply art/api/models.py]: I have changed this.

**Pull request merged**